includes/reports/overview/list-table/class-sensei-reports-overview-list-table-courses.php

```diff
@@ -279,4 +286,39 @@ protected function get_additional_filters(): array {
 			'last_activity_date_to'   => $this->get_end_date_and_time(),
 		];
 	}
+
+	/**
+	 * Get average days to completion by courses.
+	 *
+	 * @since 4.5.0
+	 *
+	 * @param array $course_ids Courses ids to filter by.
+	 * @return float Average days to completion, rounded to the highest integer.
+	 */
+	private function get_average_days_to_completion( array $course_ids ) : float {
```

Comment:
Does it make sense for this method to be a part of the new `Sensei_Reports_Overview_Service_Courses`?

Reply:
https://github.com/Automattic/sensei/pull/5091/commits/98ff8d731e122682e8821080da81d33144ccc86b